src/index.js

```diff
@@ -122,7 +121,7 @@ module.exports = exports = options => {
           send(rooms.get(key), clients, data, options))
       })
     } else {
-      send(socks.keys(), clients, data, options)
+      send(new Set([...socks.keys()]), clients, data, options)
```

Comment:
according to [node.green](http://node.green/), it will work if harmony flag is set